cla_backend/libs/eligibility_calculator/cfe_civil/age.py

```diff
@@ -0,0 +1,12 @@
+import datetime
+
+
+def translate_age(today, facts):
+    if hasattr(facts, "is_you_or_your_partner_over_60") and facts.is_you_or_your_partner_over_60:
+        age = 70
+    elif hasattr(facts, "is_you_under_18") and facts.is_you_under_18:
+        age = 17
+    else:
+        age = 50
```

Comment:
@starswan What do you think of adding a reasoning for choosing 50?

Reply:
added a comment